indico/modules/events/registration/fields/choices.py

```diff
@@ -473,6 +473,18 @@ class AccommodationField(RegistrationFormBillableItemsField):
     mm_field_class = fields.Nested
     mm_field_kwargs = {'nested': AccommodationSchema}
 
+    @property
+    def default_value(self):
+        versioned_data = self.form_item.versioned_data
+        no_accommodation_option = next(
+            (c for c in versioned_data['choices'] if c['is_no_accommodation'] and c['is_enabled']), None)
+        return {
```

Comment:
same thing here, we can use snake case since it'll be converted anyway. otherwise we have a mix between snake_case and camelCase in the `view_data` dict until the conversion happens..